web/src/main/webapp/xslt/services/sitemap/sitemap.xsl

```diff
@@ -113,17 +113,30 @@
       xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
       <xsl:for-each select="metadata/record">
         <xsl:variable name="uuid" select="uuid"/>
-        <xsl:variable name="schemaid" select="datainfo/schemaid"/>
-        <xsl:variable name="changedate" select="datainfo/changedate"/>
+        <xsl:variable name="schemaid" select="schemaid"/>
+        <xsl:variable name="changedate" select="changedate"/>
 
         <url>
           <loc>
             <xsl:choose>
               <xsl:when test="$format='xml'">
-                <xsl:value-of select="concat($nodeUrl, 'api/records/', $uuid, '/formatters/xml')"/>
+                <xsl:variable name="metadataUrlValue">
+                  <xsl:call-template name="metadataXmlDocUrl">
```

Comment:
You should keep the link to the formatters/xml in the API. Other services "xml_*" will be removed and the xml formatter is equivalent.

Reply:
Fixed, thanks